src/main/java/seedu/commands/ListCommand.java

```diff
@@ -71,4 +75,20 @@ public void execute(IncomeList incomes, SpendingList spendings) {
             Ui.printWithTab("An error occurred while listing the items.");
         }
     }
+
+    private void printSpendings(SpendingList spendings) {
+        Ui.printWithTab("Spendings");
+        Ui.printWithTab("Total spendings: " + print_list(spendings));
+        Ui.printWithTab("Daily spendings: " + spendings.getDailySpending() + " Daily Budget: " +
+                spendings.getDailyBudget());
+        Ui.printWithTab("Monthly spendings: " + spendings.getMonthlySpending() + " Monthly Budget: " +
+                spendings.getMonthlyBudget());
+        Ui.printWithTab(("Yearly spendings: " + spendings.getYearlySpending() + " Yearly Budget: " +
+                spendings.getYearlyBudget()));
+    }
+
+    private void printIncomes(IncomeList incomes) {
+        Ui.printWithTab("Incomes");
+        Ui.printWithTab("Total incomes: " + print_list(incomes));
+    }
```

Comment:
Seems to be more appropriate under ui class since it consist of only print statements